Cython/CodeWriter.py

```diff
@@ -424,29 +424,37 @@ def visit_WithStatNode(self, node):
         self._visit_indented(node.body)
 
     def visit_TryFinallyStatNode(self, node):
-        self.line(u"try:")
-        self._visit_indented(node.body)
-        self.line(u"finally:")
-        self._visit_indented(node.finally_clause)
+        self.visit(node.body)
+        self.line(u'finally:')
+        self.indent()
+        self.visit(node.finally_clause)
+        self.dedent()
 
     def visit_TryExceptStatNode(self, node):
         self.line(u"try:")
-        self._visit_indented(node.body)
+        self.indent()
+        self.visit(node.body)
+        self.dedent()
```

Comment:
Use `_visit_indented` to avoid such boilerplates.

Reply:
Thanks，modified.